assets/l10n/app_en.arb

```diff
@@ -483,5 +483,24 @@
   "notifSelfUser": "You",
   "@notifSelfUser": {
     "description": "Display name for the user themself, to show after replying in an Android notification"
+  },
+  "onePersonTyping": "{typist} is typing…",
+  "@onePersonTyping": {
+    "description": "Text to display when there is one user typing.",
+    "placeholders": {
+      "typist": {"type": "String", "example": "Alice"}
+    }
+  },
+  "twoPeopleTyping": "{typist} and {otherTypist} are typing…",
+  "@twoPeopleTyping": {
+    "description": "Text to display when there is one user typing.",
```

Comment:
This description doesn't match the text; it's also the same as the description of a different string, which makes it unhelpful for translators.

I guess https://github.com/zulip/zulip-flutter/pull/790#discussion_r1692364361 left things still confusing? The previous revision was fine on this file.

Reply:
Right, I thought https://github.com/zulip/zulip-flutter/pull/790#discussion_r1692364361 meant to have all three use this same description string.